src/System.Drawing.Primitives/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1 @@
+user32.dll!GetSysColor
```

Comment:
> Maybe it should be, but it's too late to ask now.

I'm curious - what's the rationale behind including an existing Windows API in UWP?